### ISSUE TITLE
Gerald has some earthing/shock issues that need resolving

### ISSUE THREAD

**Issue by cheapjack** [DoESLiverpool/somebody-should#1161]
@amcewen and @RussCoty got mild electric shocks touching the right side next to Sophia when cutting today with Gerald. On investigation with multimeters they seemed to find 50-70V floating on the metal side casing which was a bit worrying.

On closer inspection some wires inside the casing (the power boards and redundant fan on the right next to Sophia) appeared to have come unstuck through age. We also found a potentially weakly insulated cable from the redundant fan at the back. It's worth noting we are not sure when or if this fan is or has been working recently and did not power on during any of our live testing.

After much testing we struggled to find a real cause for this leak. We also were not fully clear how Gerald's casing is earthed to the main chassis and indeed whether the chassis is adequatly earthed. 

@RussCoty and @amcewen decided to earth the chassis with a bolt connecting chassis to base plate, directly to the mains earth entering the back of Gerald. They seem happy to have resolved the issue and have tested the casing for shocks multiple times with Gerald both on, off and during operation and have decided it is safe to use.

We will keep this issue open to log what's happened and Ive kept the bug label on for now.

It may be worth discussing this further or consider cleaning and sanity/safety checking all of Geralds electrics.

**Comment by johnmckerrell** [comment]
Out of interest does Gerald have a PAT testing sticker?

--

**Comment by cheapjack** [comment]
No but apparently it was recently PAT tested but @jackie1050 just checked and we can't seem to find a sticker but it might be in the spreadsheet for whoever did it.

**Comment by RussCoty** [comment]
So I’ve added a jumper from the earthed chassis screw at the rear/larger SMPS to the adjacent mounting chassis screw. This gives continuity across the chassis to mains Earth now.

There is a tape label to indicate the change. 

We need to check some cables that might be getting pinched by the rear  clasp on the power supply side panel. These have dropped from the adhesive clips and I fear they may have been shorting to the chassis. 

So now the circuit breaker should trip if the chassis goes live again. 

Seems like a design fault.

I’d like to see the earth added from the IEC socket rear to the chassis too at some point in a more robust manner too.

**Comment by johnmckerrell** [comment]
Maybe it was one of the things that wasn't PAT tested, I don't see it in the spreadsheet.

**Comment by cheapjack** [comment]
I used Gerald as normal successfully after the changes and I am still not on fire 😅 thanks to @amcewen & @RussCoty patient work and listening to my no-such-thing-as-stupid-questions. Sounds like we should:

- [ ] Add the earth from the IEC socket rear to the chassis
- [ ] Review all the internal cable management as most sticky cable ties have degraded. That should probably include the motherboard left side internals also but these panels are currently inaccessible
- [ ] PAT test Gerald next time we do PAT testing or after doing the above

Reckon leave this open until this is done or break out into other issues

**Comment by amcewen** [comment]
Some more detail - we had measured ~70VAC between the case of Gerald and ground (either the case of Sophia or the bandsaw).  It was only a problem when Gerald was powered up (i.e. we tried it with the blower and chiller for Gerald on and off as well as the laser-cutter itself on and off, and it was only occurring when the laser-cutter was turned on).  The voltage did drop to ~50VAC when the chiller and blower were off - maybe because the flow switch would disable some of the laser-cutter operation.  With the emergency stop activated, IIRC the voltage dropped to ~5VAC.

During investigations we didn't track down a specific problem (beyond working out the chassis wasn't earthed), but at one point the issue only recurred when we closed up the side panel (the lower one, on the right-hand-side), which is what's leading us to suspect the wiring getting pinched around there.  @RussCoty did also free up a few wires which were being pinched between the wooden board holding the power supply and the side of the case near the 36V power supply unit, so they may also have been implicated.  A visual inspection of both sets of wires didn't show any bare copper, but some more thorough checking would be good.
